projects/1.20/assets/slashblade/slashblade/lang/zh_cn.json

```
@@ -0,0 +1,179 @@
+  "adv.slashblade.quick_charge": "剑技      -迅速蓄力-",
+  "adv.slashblade.quick_charge.desc": "键位：\n攻击(♠)：%s\n指令：\n♠ 在纳刀的瞬间按下攻击 ♠",
+
+  "adv.slashblade.rapid_slash": "剑技      -迅冲斩-",
+  "adv.slashblade.rapid_slash.desc": "键位：\n攻击(♠)：%s\n前进(ᐃ)：%s\n潜行(⚪)：%s\n指令：\n地面\n⚪+ᐃ+♠",
+  "adv.slashblade.rising_star": "剑技      -星旋天翔-",
+  "adv.slashblade.rising_star.desc": "键位：\n攻击(♠)：%s\n指令：\n迅冲斩后\n♠ 按住，直到命中敌人",
+
+  "adv.slashblade.upperslash": "剑技      -上斩-",
+  "adv.slashblade.upperslash.desc": "键位：\n攻击(♠)：%s\n后退(ᐁ)：%s\n潜行(⚪)：%s\n指令：\n地面\n⚪+ᐁ+♠",
+  "adv.slashblade.upperslash_jump": "剑技      -跃升斩-",
+  "adv.slashblade.upperslash_jump.desc": "键位：\n攻击(♠)：%s\n指令：\n上斩后\n♠ 按住",
+
+
+  "adv.slashblade.air_trick":   "能力      -隔空瞬步-",
+  "adv.slashblade.air_trick.desc": "向注视中的敌人的头上瞬间移动\n键位：\n疾跑(♠)：%s\n前进(ᐃ)：%s\n潜行(⚪)：%s\n指令：\n⚪+ᐃ+♠",
+  "adv.slashblade.trick_down":  "能力      -瞬步退行-",
+  "adv.slashblade.trick_down.desc": "滞空时迅速落地\n键位：\n疾跑(♠)：%s\n后退(ᐁ)：%s\n潜行(⚪)：%s\n指令：\n⚪+ᐁ+♠",
+  "adv.slashblade.trick_dodge": "能力      -瞬步回避-",
+  "adv.slashblade.trick_dodge.desc": "往任意方向瞬间移动躲避伤害\n键位：\n疾跑(♠)：%s\n指令：\n在移动中 ♠",
+  "adv.slashblade.trick_up":    "能力      -瞬步上行-",
+  "adv.slashblade.trick_up.desc": "向上瞬间移动\n键位：\n疾跑(♠)：%s\n前进(ᐃ)：%s\n潜行(⚪)：%s\n指令：\n⚪+ᐃ+♠",
+  "adv.slashblade.enemy_step":  "能力      -借力跳-",
+  "adv.slashblade.enemy_step.desc": "滞空时把敌人作为垫脚石以二段跳\n键位：\n跳跃(□)：%s\n指令：\n滞空靠近敌人时 □",
+  "adv.slashblade.kick_jump":   "能力      -踢跃-",
+  "adv.slashblade.kick_jump.desc": "蹬开墙壁再度跳跃\n键位：\n跳跃(□)：%s\n指令：\n滞空靠近墙壁时 □",
+
+  "adv.slashblade.guard":       "能力      -防御-",
+  "adv.slashblade.guard.desc": "消耗耐久度将敌人的远程攻击无效\n'行走'中无法发动\n键位：\n潜行(⚪)：%s\n指令：\n⚪",
+  "adv.slashblade.guard_just":  "能力      -格挡-",
+  "adv.slashblade.guard_just.desc": "将敌人的攻击无效\n高评价状态中不会消耗耐久度\n评价会上升\n键位：\n潜行(⚪)：%s\n指令：\n在受到攻击时 ⚪",
+
+  "adv.slashblade.summonedswords": "射技      -幻影剑-",
+  "adv.slashblade.summonedswords.desc": "消耗1点经验值，射出幻影剑\n键位：\n%s(♠)：%s\n指令：\n♠",
+  "adv.slashblade.spiral_swords":  "射技      -环形幻剑阵-",
+  "adv.slashblade.spiral_swords.desc": "消耗5点经验值，在自身周围形成幻影剑圆阵\n键位：\n%s(♠)：%s\n指令：\n♠ 按住",
+  "adv.slashblade.storm_swords":   "射技      -怒风幻剑阵-",
+  "adv.slashblade.storm_swords.desc": "消耗5点经验值，将幻影剑如同怒风般包围攻击并击飞敌人\n键位：\n%s(♠)：%s\n后退(ᐁ)：%s\n潜行(⚪)：%s\n指令：\n⚪+ᐁ+♠ 按住",
+  "adv.slashblade.blistering_swords": "射技      -急袭幻剑阵-",
+  "adv.slashblade.blistering_swords.desc": "消耗5点经验值，将幻影剑展开快速攻击敌人\n键位：\n%s(♠)：%s\n前进(ᐃ)：%s\n潜行(⚪)：%s\n指令：\n⚪+ᐃ+♠ 按住 放开",
+  "adv.slashblade.heavy_rain_swords": "射技      -暴雨幻剑阵-",
+  "adv.slashblade.heavy_rain_swords.desc": "消耗5点经验值，将幻影剑如同五月雨般在敌人头上降下\n键位：\n%s(♠)：%s\n后退(ᐁ)：%s\n前进(ᐃ)：%s\n潜行(⚪)：%s\n指令：\n⚪+ᐁ to ᐃ+♠ 按住",
+
+
+
+  "adv.slashblade.exeffect": "附魔增效",
+  "adv.slashblade.exeffect.desc": "附魔提供额外的效果提升",
+
+  "adv.slashblade.exeffect.soul_speed": "灵魂疾行",
+  "adv.slashblade.exeffect.soul_speed.desc": "灵魂疾行每1级能够延长格挡指令输入窗口1tick",
+
+  "adv.slashblade.exeffect.feather_falling": "缓降",
+  "adv.slashblade.exeffect.feather_falling.desc": "空中行动时降落速度收到缓降效果等级的影响"

```

```suggestion
  "adv.slashblade.exeffect.feather_falling.desc": "缓解效果的等级将减缓空中行动的下落速度"
```
`空中`行动建议和出招表统一（这里可以替换为`滞空`，不过最好还是统一